src/index.ts

```diff
@@ -215,22 +212,18 @@ export async function waitRunDependencies(Module: any): Promise<void> {
 
 export async function solve(
   yml: string,
-  logger?: ILogger,
-  locateWasm?: (file: string) => string
+  logger?: ILogger
 ): Promise<{ condaPackages: ISolvedPackages; pipPackages: ISolvedPackages }> {
-  const picomamba = await initEnv(logger, locateWasm);
+  const condaPackages = (await getSolvedPackages(
+    yml,
+    logger
+  )) as ISolvedPackages;
 
-  const condaPackages = (await picomamba.solve(yml)) as ISolvedPackages;
   let pipPackages: ISolvedPackages = {};
 
-  logger?.log('');
```

Comment:
Let's keep this `logger?.log('');` around, it was giving a bit of space in the logs.